qbits_backend_demo.py

```diff
@@ -0,0 +1,29 @@
+from auto_round import AutoRound
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+model_name = "facebook/opt-125m"
+path = "/home/wangzhe/.cache/huggingface/hub/models--facebook--opt-125m/snapshots/27dcfa74d334bc871f3234de431e71c6eeba5dd6"
+model = AutoModelForCausalLM.from_pretrained(
+    path, torch_dtype="auto", trust_remote_code=True)
+tokenizer = AutoTokenizer.from_pretrained(path, trust_remote_code=True)
+
+
+bits, group_size, sym = 4, 128, False
+# device:Optional["auto", None, "hpu", "cpu", "cuda"]
+autoround = AutoRound(model, tokenizer, bits=bits, group_size=group_size,
+                      sym=sym, nsamples=32, seqlen=32, device="cpu", iters=1, scale_dtype="fp32")
```

Comment:
the default scale_dtype is fp16, is it the must for qbits? please remove this file before the pr merging

Reply:
yes, already changed to default value.